### PR TITLE
Rework holland commvault wrapper

### PR DESCRIPTION
**Pull request by soulen3**
- Lint files
- Add holland_commvault module to pylint list in Travis CI
- Add logic so the commvault wrapper will wait for another instance of the wrapper to complete, and then return the other processes return code. This uses the commvault 'job' id, which is expected to be different for distant runs and backup sets.

(requires pid module)

### CODE REVIEW

File: contrib/holland-commvault/holland_commvault/commvault.py [hunk 1]
```diff
@@ -94,7 +105,35 @@ def main():
...
+            return ret
+    except PidFileAlreadyLockedError:
+        pid_file = open(pid_location, 'r')
+        pid = pid_file.read()
+        pid_file.close()
+
+        logging.info("Holland (commvault agent) is already running, waiting for the pid %s", pid)
+        while os.path.isfile(pid_location):
+            sleep(5)
+        try:
```
Comment (nwbreneman) [inline contrib/holland-commvault/holland_commvault/commvault.py]: Can we tighten up the exception handling for this section? (The outer try/except looks fine to me.)

i.e. not wrapping as much within the try block and not catching BaseException (or Exception) unless absolutely necessary; example:

```
try:
    status = open(status_file, 'r')
    ret = int(status.read())
except IOError:
    logging.info("Holland (commvault agent) failed to open/read status file")
    return 1
else:
    status.close()
    return ret
```

or maybe:

```
try:
    with open(status_file, 'r') as status:
        ret = int(status.read())
except IOError:
    logging.info("Holland (commvault agent) failed to read status file")
    return 1
else:
    return ret
```

to give us the functionality of a `finally` statement to make sure the file handle's always closed if we can open it but not read it. We can nest another try block to give more detailed logging on the failure if you want as well.

File: contrib/holland-commvault/holland_commvault/commvault.py [hunk 2]
```diff
@@ -94,7 +105,35 @@ def main():
...
+            status.write(str(ret))
+            status.close()
+            return ret
+    except PidFileAlreadyLockedError:
+        pid_file = open(pid_location, 'r')
+        pid = pid_file.read()
+        pid_file.close()
+
+        logging.info("Holland (commvault agent) is already running, waiting for the pid %s", pid)
+        while os.path.isfile(pid_location):
```
Comment (nwbreneman) [inline contrib/holland-commvault/holland_commvault/commvault.py]: Do we want to set a reasonable timeout for this? I'm not sure if there's a scenario where commvault would run for so long that we'd want Holland to error out waiting for it and maybe cause a failure so someone looks into it.

File: contrib/holland-commvault/holland_commvault/commvault.py (hunk 1 above)
Comment (soulen3) [reply contrib/holland-commvault/holland_commvault/commvault.py]: I've update the error handling to what you recommended here

File: contrib/holland-commvault/holland_commvault/commvault.py (hunk 2 above)
Comment (soulen3) [reply contrib/holland-commvault/holland_commvault/commvault.py]: I've added a 10 hour timeout

**Pull request merged**